### PR TITLE
Print matching files

### PR DESCRIPTION
**Pull request by Schniz**
For allowing nice shell piping, like:

```
vim $(globitor) # Open files with Vim
# or
code $(globitor) # Open files with VSCode
```

### CODE REVIEW

File: src/cli.ts [hunk 1]
```diff
@@ -52,9 +54,15 @@ interactive({
   .then((result) => {
     result.pattern = result.pattern || '**';
     clipboardy.writeSync(result.pattern);
-    console.log(`Pattern: ` + chalk.cyan(result.pattern));
-    console.log(`Matched: ` + result.files.length);
-    console.log(chalk.yellow.dim`Copied pattern to clipboard!`);
+    console.error(`Pattern: ` + chalk.cyan(result.pattern));
+    console.error(`Matched: ` + result.files.length);
+    console.error(chalk.yellow.dim`Copied pattern to clipboard!`);
```
Comment (Schniz) [inline src/cli.ts]: Using `console.error` will write to `stderr` which is an unfortunate name, but it is file descriptor 2, and not the "default" one. So `globitor | xargs rm` for instance will still print `Pattern: ... Matched: ...` to the screen, and will remove the files. 😃

**Comment by Schniz** [comment]
sorry it _does not_ allow it. we need that all of the UI will be printed in stderr as well 👿

**Comment by Schniz** [comment]
Fixed

**Review by ranyitz — COMMENTED** [review]
That's a cool usage! Haven't thought of it

I don't want to introduce flags, so here is my suggestion:

1. Always print the selected files (no need for `--silent`)
2. Copying to the clipboard might be an unwanted side-effect in this flow, so it sounds reasonable to remove this capability.
3.  Make it looks nice for the non vim use-case.

I'll create a PR to this PR with the those changes

**Pull request merged**